src/renderer/widgets/PasswordInputField.cpp

```diff
@@ -176,18 +179,18 @@ bool CPasswordInputField::draw(const SRenderData& data) {
 
     failedAttempts = g_pHyprlock->getPasswordFailedAttempts();
```

Comment:
I've added this and check above to force draw on empty input, because nothing was triggering it and failasset was missing.
Here is [fixup.patch](https://github.com/hyprwm/hyprlock/files/14652880/fixup.patch) to use it directly in placeholder function, because now failasset is missing when typing directly after failure (without showing normal asset) and fail again, and also on empty input.

It also unbreaks some color changing.

Reply:
Thanks for the patch! :)